src/bin-support.js

```diff
@@ -12,7 +12,9 @@ async function runTransform(binRoot, transformName, args, extensions = DEFAULT_E
   let { paths, options } = parseTransformArgs(args);
 
   try {
-    let foundPaths = await globby(paths);
+    let foundPaths = await globby(paths, {
+      expandDirectories: { extensions: extensions.split(','), }
```

Comment:
Some linting failures here:

```
/home/travis/build/rwjblue/codemod-cli/src/bin-support.js
  16:61  error  Replace `,·}` with `·},`  prettier/prettier
```

Should be fixed with a `yarn eslint . --fix`

Reply:
Sorry for that, it should be good now.